modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/IgniteRocksDbException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.storage.rocksdb;
+
+import static org.apache.ignite.lang.ErrorGroups.Storage.STORAGE_CORRUPTED_ERR;
+
+import org.apache.ignite.internal.storage.StorageException;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Exception that gets thrown if an error occurs in the underlying RocksDB storage.
+ */
+public class IgniteRocksDbException extends StorageException {
+    public IgniteRocksDbException(@Nullable Throwable cause) {
+        super(STORAGE_CORRUPTED_ERR, cause);
```

Comment:
It looks like the error code here is not quite appropriate, maybe we will make a general error code for now, and when we determine what kind of error it is, we will be more specific?

Reply:
Can you elaborate please?

Reply:
At the moment we are more or less able to detect data corruption only in our "page-memory" engine. In rocks, as I understand it, we are unable to distinguish data corruption errors from, for example, closing the storage and trying to read from it, and we interpret any errors from rocks as data corruption. This looks a bit strange to me. It is better to find cases of data corruption errors and highlight them.

Reply:
I agree, but as I understand there's no way we could distinguish it. Maybe we should just introduce a more broad error code, that will not only be about data corruption?

Reply:
Seems reasonable.